client/components/language-picker/modal.jsx

```diff
@@ -70,23 +71,20 @@ export class LanguagePickerModal extends PureComponent {
 		useFallbackForIncompleteLanguages: false,
 	};
 
-	constructor( props ) {
-		super( props );
-
-		this.state = {
-			filter: getLanguageGroupByCountryCode( this.props.countryCode ),
-			showingDefaultFilter: true,
-			search: false,
-			isSearchOpen: false,
-			selectedLanguageSlug: this.props.selected,
-			suggestedLanguages: this.getSuggestedLanguages(),
-			empathyMode: props.empathyMode,
-			useFallbackForIncompleteLanguages: props.useFallbackForIncompleteLanguages,
-		};
-
-		this.languagesList = React.createRef();
-		this.selectedLanguageItem = React.createRef();
-	}
+	state = {
+		empathyMode: this.props.empathyMode,
+		filter: getLanguageGroupByCountryCode( this.props.countryCode ),
+		isSearchOpen: false,
+		search: false,
+		selectedLanguageSlug: this.props.selected,
+		showingDefaultFilter: true,
+		suggestedLanguages: this.getSuggestedLanguages(),
+		useFallbackForIncompleteLanguages: this.props.useFallbackForIncompleteLanguages,
+	};
+
+	initiallySelectedLanguageSlug = this.props.selected;
```

Comment:
Any chance `this.props.selected` won't be set during the initial mounting of the component?

Reply:
`selected` defaults to `en` here: https://github.com/Automattic/wp-calypso/blob/56a4a67a9a692eedd5625b90b9cbc1ae5feb570e/client/components/language-picker/modal.jsx#L67